zhusuan/hmc.py

```diff
@@ -460,6 +460,11 @@ def iss():
                     self.q, p, current_q, current_p,
                     get_log_posterior, mass, self.data_axes)
 
+            old_log_prob = tf.check_numerics(old_log_prob, 'HMC: old_log_prob is NaN! Try better initialization?')
```

Comment:
line width, and numeric errors not limited to nan, also inf/very large numbers. The numeric value will be raised in internal message. So we can just say "has numeric errors"